.github/workflows/release-e2e-workflow-template.yml

```diff
@@ -78,7 +74,8 @@ jobs:
         with:
           working-directory: cypress-test
           command: ${{ inputs.test-command }}
-          wait-on: 'http://localhost:5601'
+          start: ${{ steps.get_opensearch.RUN_CMD }}, ${{ steps.get_opensearch_dashboards.RUN_CMD }}
+          wait-on: 'http://localhost:9200, http://localhost:5601'
```

Comment:
In bundled OSD (with security enabled), the 9200 is under `https` not `http`.

Reply:
shoot good catch